new/t2.ts

```diff
@@ -0,0 +1,18 @@
+var x = null; // error
+        ~~~~           [0]
+console.log(null, x); // error
+            ~~~~               [0]
+
+let match: string | null;
+interface foo {
+    bar: [number, null, string];
+}
+
+if (document.querySelector('.foo') === null) {}
+if (document.querySelector('.foo') == null) {}
+                                      ~~~~ [0]
+if (null !== document.querySelector('.foo')) {}
+if (null != document.querySelector('.foo')) {}
```

Comment:
Use 'undefined' instead of 'null'

```suggestion
if (undefined != document.querySelector('.foo')) {}
```